node/engine/store/memstore.go

```diff
@@ -24,6 +24,7 @@ type MemStore struct {
 	consensusChannels  safesync.Map[[]byte]
 	channelToObjective safesync.Map[protocols.ObjectiveId]
 	vouchers           safesync.Map[[]byte]
+	blocks             safesync.Map[uint64]
```

Comment:
I agree that's overkill in this situation. Changed in https://github.com/statechannels/go-nitro/pull/1581/commits/ca4f034bbff090d1d34ddf1c90a970e36d318593

Reply:
Cool. I suppose by that change does render that variable "concurrency unsafe" in theory, since there is no longer any `sync.Mutex` under the hood preventing concurrent reads/writes. Might be worth running the race condition checker on this branch and/or adding a mutex if we think that is necessary.

Reply:
Also, do we want to make the similar change in the `durablestore`, too?